src/Administration/Login/UserService/UserService.php

```diff
@@ -0,0 +1,173 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Administration\Login\UserService;
+
+use Doctrine\DBAL\Connection;
+use Shopware\Administration\Login\Exception\LoginException;
+use Shopware\Administration\Login\TokenService\ParsedIdToken;
+use Shopware\Administration\Login\TokenUserCollection;
+use Shopware\Administration\Login\TokenUserDefinition;
+use Shopware\Core\Framework\DataAbstractionLayer\EntityRepository;
+use Shopware\Core\Framework\Log\Package;
+use Shopware\Core\Framework\RateLimiter\Exception\RateLimitExceededException;
+use Shopware\Core\Framework\RateLimiter\RateLimiter;
+use Shopware\Core\Framework\Uuid\Uuid;
+use Shopware\Core\System\User\UserCollection;
+use Shopware\Core\System\User\UserDefinition;
+
+/**
+ * @internal
+ */
+#[Package('after-sales')]
+final class UserService
```

Comment:
see above, we need a more descriptive name as we already have the name `User` in a lot of places